omero/users/cli/import-bulk.rst

```diff
@@ -0,0 +1,99 @@
+Using bulk imports
+==================
+
+The CLI import option ``--bulk`` specifies a configuration file that
+can be used to perform a batch of imports with the same or similar
+options. The file is written in a simple YAML syntax and can be placed
+anywhere.
+
+A simple YAML file might look like: ::
+
+    ---
+    continue: "true"
+    transfer: "ln_s"
```

Comment:
Another question during the Basel workshop was:

  - how can I use it without the in-place option ?
 
Although it might be argued that this is intuitive (remove the ..ln_s line), it would be still nicer if we would explicitly suggest it (or even not start from a strong silent assumption that in-place is a default here - might make more sense, but obviously users want the full range of options)

Reply:
agree that we should not indicate transfer is the "default"
also since no files are moved having checksum is confusing for some

Reply:
I'll try an absolute minimal example (i.e. just path) at the beginning and build up from there.